sample-model/build.gradle

```diff
@@ -58,6 +58,9 @@ dependencies {
 
 gradle.projectsEvaluated {
     tasks.withType(JavaCompile) {
-        options.compilerArgs << "-Xlint:all,-deprecation,-serial,-processing" << "-Werror"
+        options.compilerArgs += [
```

Comment:
Only `sample-java-model` is utilizing the ltgt plugin (it's on the latest), the rest use the android annotation processor stuff, this block just turns on java lint all the way up and sets warnings as errors so that any bad generated code causes the build to fail 👍